docs/attributes-registry/gen-ai.md

```diff
@@ -13,34 +13,28 @@
+| Attribute                          | Type     | Description                                                                                      | Examples                  | Stability                                                        |
+| ---------------------------------- | -------- | ------------------------------------------------------------------------------------------------ | ------------------------- | ---------------------------------------------------------------- |
+| `gen_ai.operation.name`            | string   | The name of the operation being performed. [1]                                                   | `chat`; `text_completion` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.frequency_penalty` | double   | The frequency penalty setting for the GenAI request.                                             | `0.1`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.max_tokens`        | int      | The maximum number of tokens the model generates for a request.                                  | `100`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.model`             | string   | The name of the GenAI model a request is being made to.                                          | `gpt-4`                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.presence_penalty`  | double   | The presence penalty setting for the GenAI request.                                              | `0.1`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.stop_sequences`    | string[] | List of sequences that the model will use to stop generating further tokens.                     | `["forest", "lived"]`     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.temperature`       | double   | The temperature setting for the GenAI request.                                                   | `0.0`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.top_k`             | double   | The top_k sampling setting for the GenAI request.                                                | `1.0`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.request.top_p`             | double   | The top_p sampling setting for the GenAI request.                                                | `1.0`                     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gen_ai.response.finish_reasons`   | string[] | Array of reasons the model stopped generating tokens, corresponding to each generation received. | `["stop"]`                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
> consider adding tool_calls to finish_reasons or explain why not

finish_reasons is an array of a free form strings - we didn't get to defining a unified list yet (since this spec is in early development). If you check examples, you'll find finish_reason=tool_call used there.

So there is no place to add it to (yet). This PR does not change anything around it. I'll happily review a PR which normalizes a list of such reasons, but won't add it to the scope of this PR.